src/annotator/adder.js

```diff
@@ -134,6 +142,24 @@ export class Adder {
     this._render();
   }
 
+  /**
+   * Display the adder in the best position in order to target the
+   * selected text in `targetRect`.
+   *
+   * @param {DOMRect} targetRect - The rect of text to target, in viewport
+   *        coordinates.
+   * @param {boolean} isSelectionBackwards - True if the selection was made
```

Comment:
Just a random note: this assumes all displays are rendered left-to-right

Reply:
Could you let me know what make you think this assumes left-to-right?

Reply:
If backwards means selecting right-to-left...that implies that the display is assumed to be left-to-right...

Maybe the word "backwards" is misleading here :)